api-reference/10 UI Widgets/dxDataGrid/3 Methods/exportToExcel(selectionOnly).md

```diff
@@ -1,6 +1,6 @@
 ---
 id: dxDataGrid.exportToExcel(selectionOnly)
-deprecated: 
+deprecated: Since v20.1, we recommend using [ExcelJS-based export](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/) which does not use this option.
```

Comment:
This method has a direct alternative - `exportDataGrid`. @EugeniyKiyashko we need to specify this in the doccomment.

Reply:
Oh, certainly. But I can't fixing it now. I created a Trello card so as not to forget to fix it later.